CK3toEU4/Data_Files/blankMod/output/common/government_reforms/ZZZ_enhanced_converter_reforms_feudal.txt

```diff
@@ -522,7 +519,7 @@ mughal_government = {
 			province_warscore_cost = -0.2
 		}
 		scythian_group = {
-			cav_to_inf_ratio = 0.1
+			cavalry_fire = 1
```

Comment:
Any fire=1 modifier is insanely powerful early game as fire values are really low. To the point where 3-4 cavs will be able to stackwipe 5x larger armies. I'd suggest cavalry combat ability +10% instead.

Reply:
Is it that much stronger than Spanish having Artiller fire +1?

Reply:
In their national ideas